consoleme/lib/slack.py

```diff
@@ -117,3 +101,35 @@ async def get_payload(
         ]
     }
     return payload
+
+
+async def send_slack_notification(payload, payload_id):
+    """
+    Sends a notification using specified webhook URL about a new request created
+    """
+    if not config.get("slack.notifications_enabled", False):
+        log.info(f"Slack notifications not enabled. Not sending payload: {payload_id}")
+        return
+
+    slack_webhook_url = config.get("slack.webhook_url")
+    if not slack_webhook_url:
+        log.error(f"Missing webhook URL for slack notification. Not sending payload: {payload_id}")
+        return
+
+    http_headers = HTTPHeaders({"Content-Type": "application/json"})
+    http_req = HTTPRequest(
+        url=slack_webhook_url,
+        method="POST",
+        headers=http_headers,
+        body=json.dumps(payload),
+    )
+
+    http_client = AsyncHTTPClient(force_instance=True)
+    try:
+        await http_client.fetch(request=http_req)
+        log_data["message"] = "Slack notification sent"
```

Comment:
This dict needs to be initialized prior to this

Reply:
I logged without a dictionary

Reply:
Even better!